src/common/registry-csm/main/constants.ts

```diff
@@ -0,0 +1 @@
+export const REGISTRY_GLOBAL_OPTIONS_TOKEN = Symbol('registryGlobalOptions');
```

Comment:
it looks unsafe use the same value as in  registry

Reply:
You are right that this can be a problem, but in our case we are using Symbol and Symbol has the property that Symbol(‘registryGlobalOptions’) !== Symbol(‘registryGlobalOptions’)